packages/react-examples/src/react/Nav/Nav.Wrapped.Example.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { Nav, INavStyles, INavLinkGroup } from '@fluentui/react/lib/Nav';
+
+const navStyles: Partial<INavStyles> = {
+  root: {
+    width: 208,
+    height: 350,
+    boxSizing: 'border-box',
+    border: '1px solid #eee',
+    overflowY: 'auto',
+  },
+  link: {
+    whiteSpace: 'normal',
+    lineHeight: 'inherit',
+  },
+};
+
+const navLinkGroups: INavLinkGroup[] = [
+  {
+    links: [
+      {
+        name: 'Home',
+        url: 'http://example.com',
+        expandAriaLabel: 'Expand Home section',
+        collapseAriaLabel: 'Collapse Home section',
+        title: '',
```

Comment:
Probably also good to comment on the first blank title

Reply:
I'll add one comment to the top of the block with the reasoning 👍